blivet/devices/__init__.py

```diff
@@ -19,7 +19,7 @@
 # Red Hat Author(s): David Lehman <dlehman@redhat.com>
 #
 
-from .lib import device_path_to_name, device_name_to_disk_by_path, ParentList
+from .lib import device_path_to_name, device_name_to_disk_by_path, ParentList, Tags
```

Comment:
I'll remove it. I think I initially expected that it would be useful to have them importable from `devices`, but I can't remember why.